src/lucky/param_helpers.cr

```diff
@@ -1,7 +1,5 @@
 module Lucky::ParamHelpers
-  @_params : Lucky::Params?
-
-  def params : Lucky::Params
-    @_params ||= Lucky::Params.new(context.request, @route_params)
+  memoize def params : Lucky::Params
+    context.params
   end
```

Comment:
This is basically just so you can still call `params` in your actions without having to call `context.params`.